### ISSUE TITLE
Add support for quotes

### ISSUE THREAD

**Issue by himynameistimli** [freakboy3742/pyxero#274]
Support for quotes was added a few days back, with a documentation update https://developer.xero.com/documentation/api/quotes.

Just browsing through, it seems similar to Invoices, so it shouldn't be too difficult to add on.  Just note that not all of the documentation has been updated, I still can't find the the proper values for `QuoteStatus`

![image](https://user-images.githubusercontent.com/6541272/69316945-cdfef800-0c74-11ea-8384-33756888a7d0.png)

**Comment by bpotard** [comment]
Try this:
```patch
diff --git a/xero/api.py b/xero/api.py
index 4331daa..bd49b48 100644
--- a/xero/api.py
+++ b/xero/api.py
@@ -30,6 +30,7 @@ class Xero(object):
         "Payments",
         "Prepayments",
         "PurchaseOrders",
+        "Quotes",
         "Receipts",
         "RepeatingInvoices",
         "Reports",
```

Seems to work fine!

**Comment by freakboy3742** [comment]
This was included as part of v0.9.2.
